test/index.js

```diff
@@ -152,6 +153,26 @@ describe('KISSmetrics', function () {
       result['Billing Amount'].should.eql(19.95);
       result._n.should.eql('Baked a cake');
     });
+
+    it('should use a POST request if the payload exceeds a threshold', function(done){
+      var largeEvent = helpers.repeat('x', 2100);
+      var timestamp = Math.floor(Date.now() / 1000);
+
+      test
+        .set(settings)
+        .track({event: largeEvent, time: timestamp})
```

Comment:
> My only reasoning against putting the large payload in a JSON file is that it's not clear that a >2KB string is >2KB. I think it's clearer generating the string from within the test function.

I talked with @f2prateek about this and we'd rather keep the convention and use a fixture. You could name it `fixture-over-2kb` and just add enough properties/traits in it and locally check the file size to make sure if it is over 2kb.